openreview/profile/management.py

```diff
@@ -287,73 +295,157 @@ def set_archive_invitations(self):
+                                    }
+                                }
+                            },                        
+                            'abstract': {
+                                'order': 4,
+                                'description': 'Abstract of paper. Add TeX formulas using the following formats: $In-line Formula$ or $$Block Formula$$.',
+                                'value': {
+                                    'param': {
+                                        'type': 'string',
+                                        'maxLength': 5000,
+                                        'markdown': True,
+                                        'input': 'textarea'
+                                    }
+                                }
+                            },
+                            'pdf': {
+                                'order': 5,
+                                'description': 'Upload a PDF file that ends with .pdf.',
+                                'value': {
+                                    'param': {
+                                        'type': 'file',
+                                        'maxSize': 50,
+                                        'extensions': ['pdf'],
+                                        'optional': True
+                                    }
+                                }
+                            },
+                            'html': {
+                                'order': 6,
+                                'description': 'Enter a URL to a PDF file.',
+                                'value': {
+                                    'param': {
+                                        'type': 'string',
+                                        'regex': '"(http|https):\\/\\/.+"',
+                                        'optional': True
+                                    }
+                                }
+                            },
+                            'venue': {
+                                'order': 7,
+                                'description': 'Enter the venue where the paper was published.',
+                                'value': {
+                                    'param': {
+                                        'type': 'string',
+                                        'optional': True
+                                    }
+                                }
+                            },
+                            'year': {
+                                'order': 8,
+                                'description': 'Enter the year where the paper was published.',
+                                'value': {
+                                    'param': {
+                                        'type': 'integer',
+                                        'optional': True
+                                    }
+                                }
+                            },
```

Comment:
We are going to ask for this here instead of using pdate? Is it because the note editor only allows to edit fields inside the note content?

Reply:
and we don't have a date picker to choose the pdate but Xukun implemented one. Do you think we should ask a pdate instead?

Reply:
cc @xkopenreview to see if we could use the date picker in the note editor to select the pdate.

Reply:
> and we don't have a date picker to choose the pdate but Xukun implemented one. Do you think we should ask a pdate instead?

That's what we are asking in the content of the note right?

Reply:
is direct upload the only place where pdate is user entered?

Reply:
so far, yes, this is the first case when we need to ask the user to enter the pdate.

Reply:
pdate is implemented now.